www/qserv/js/ToolsSql.js

```diff
@@ -0,0 +1,219 @@
+define([
+    'webfwk/CSSLoader',
+    'webfwk/Fwk',
+    'webfwk/FwkApplication',
+    'underscore'],
+
+function(CSSLoader,
+         Fwk,
+         FwkApplication) {
+
+    CSSLoader.load('qserv/css/ToolsSql.css');
+
+    class ToolsSql extends FwkApplication {
+
+        constructor(name) {
+            super(name);
+        }
+
+        /**
+         * Override event handler defined in the base class
+         *
+         * @see FwkApplication.fwk_app_on_show
+         */
+        fwk_app_on_show() {
+            console.log('show: ' + this.fwk_app_name);
+            this.fwk_app_on_update();
+        }
+
+        /**
+         * Override event handler defined in the base class
+         *
+         * @see FwkApplication.fwk_app_on_hide
+         */
+        fwk_app_on_hide() {
+            console.log('hide: ' + this.fwk_app_name);
+        }
+
+        /**
+         * Override event handler defined in the base class
+         *
+         * @see FwkApplication.fwk_app_on_update
+         */
+        fwk_app_on_update() {
+            if (this.fwk_app_visible) {
+                this._init();
+            }
+        }
+
+        /**
+         * The first time initialization of the page's layout
+         */
+        _init() {
+            if (this._initialized === undefined) {
+                this._initialized = false;
+            }
+            if (this._initialized) return;
+            this._initialized = true;
+
+            let html = `
+<div id="fwk-tools-sql">
+  <div class="row">
+    <div class="col">
+      <p>This tool allows executing queries directly against the <span style="font-weight:bold;">MariaDB</span>
+        service of a <span style="font-weight:bold;">Qserv</span> worker. For queries which produce a result set
+        the result will be displayed as a table below the form. Errors will be also reported.
+        Please, <span style="font-weight:bold;">DO NOT</span> launch queries which may produce millions of rows.
+        An attept to do so will make thsi applicatio unresponsive.
```

Comment:
attept thsi applicatio 

It already feels half-unresponsive, some characters are lost 😉

Reply:
Noted! Will fix it.